src/assets/sass/variables/colors.scss

```
@@ -57,6 +57,8 @@ $graphite-black: #1c1c1c;
 $black: #000000;
 $donate-tooltip-color: #333333;
 
+$overlay-bg: #2f424b;
```

Stylelint (17.14.0)

[error] Unexpected empty line before $-variable (scss/dollar-variable-empty-line-before)

(scss/dollar-variable-empty-line-before)